### PR TITLE
[No Ticket][Risk=low] Abort running API tests if no api changes and not master

### PR DESCRIPTION
**Pull request by blrubenstein**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -22,6 +22,13 @@ jobs:
       - checkout
       - run:
           command: git submodule update --init --recursive
+      - run:
+        #Circle is bad about setting up your remotes for you, you must use origin/
+          command: |
+            if [ $(git diff --name-only $(git merge-base origin/master ${CIRCLE_BRANCH}) | grep api/ | wc -l | xargs) == 0 ] && [ ${CIRCLE_BRANCH} != "master" ]; then
```
Comment (calbach) [inline .circleci/config.yml]: I would reverse the order of these predicates. I would also include the check for empty CIRCLE_BRANCH (I believe this happens when we tag a release - not 100%)

File: .circleci/config.yml [hunk 2]
```diff
@@ -22,6 +22,13 @@ jobs:
       - checkout
       - run:
           command: git submodule update --init --recursive
+      - run:
+        #Circle is bad about setting up your remotes for you, you must use origin/
```
Comment (calbach) [inline .circleci/config.yml]: nit: +2 indent

File: .circleci/config.yml (hunk 1 above)
Comment (blrubenstein) [reply .circleci/config.yml]: Will CIRCLE_BRANCH being empty be caught by != master?

Comment (blrubenstein) [reply .circleci/config.yml]: Will do!

**Comment by calbach** [comment]
We want to run tests when we do a release

**Comment by blrubenstein** [comment]
Yup, I realized right after commenting what you were actually saying... Should be good now @calbach.

**Pull request merged**